config/locales/en.yml

```diff
@@ -17,3 +17,12 @@ en:
 
   titles:
     application: Budgetapp
+
+  activerecord:
+    errors:
+      models:
+        budget:
+          attributes:
+            name:
+              blank: "must be entered"
+              taken: "already taken"
```

Comment:
You can use a more generic format for the yams structure, to remove the duplication in error messages.
Check the module on translations in the curriculum.

Reply:
Thanks Elle, will look at this and the factory above - great suggestions!
Ignore in the next PRs as I will take this on for all models!